### DISCUSSION TITLE
Modelo de contratação 100% remoto

### DISCUSSION THREAD

**Discussion by mcarneiro** [frontendbr/forum#1105] [Discussão]
Pessoal, to dando uma estudada em modelos de contratação 100% remoto na área de front-end e queria bater um papo com quem já trabalhou nesse formato pra entender alguns aspectos, principalmente do quão maduro o formato está no Brasil.

Algumas questões já de cara (pra quem quiser responder nos comentários):

- Existiu plano de carreira ou era só esquema "freela" / braço de desenvolvimento?
- Ou melhor, alguém teve a experiência de ter uma equipe trabalhando remoto?
- A empresa dava ajuda de custo para internet / máquina / software?
- Relacionamento era 100% virtual ou existia eventos presenciais entre as pessoas de diferentes regiões? Se sim, imagino que empresa pagou passagem e estadia, certo?
- Alguém teve a experiência sendo contratado CLT?
- Salário era baseado no custo de vida do local do empregado ou da empresa? (o clássico conto do "uou, trabalho em Lençóis, no interior da Bahia com o salário de São Paulo!")
- Haviam benefícios como saúde, alimentação, creche, etc? Se sim, como funcionavam? Ajuda de custo, convênio, etc?
- Existiu senso de equipe? Ou sentimento de que o time alocado tomava decisões sem consultar os remotos (já passei por isso trabalhando com uma empresa polonesa).

Quem puder e preferir puxar um papo via inbox, me adicionem e depois comento com os outputs :)

**Comment by cjpatoilo** [comment]
@mcarneiro um "survey" não captaria melhor essas informações? Vou tentar responder tudo e se ficar algumas dúvida você pode ficar a vontade p/ me adicionar e trocar uma ideia. Basta procurar por @cjpatoilo em qualquer rede social ;)

Tenho ~6 anos na área Front-End e apenas meus primeiros ~2 anos de carreira não foram remotos. Atualmente trabalho remoto para uma empresa sendo o único 100% remoto. Já houve outro 100% remoto além de mim e vez ou outra alguns do team viajam para outras cidades e acabam trabalhando remoto. O formato de contratação é CLT com praticamente todos auxílios e benefícios competitivos que uma empresa comum oferece assim como você citou. O Salário é baseado nos custos de vida do local da empresa sendo o mesmo valor para todos os desenvolvedores. Minha especialidade é Front-End, porém temos a oportunidade de desenvolver habilidades em outras pontas. Atualmente venho estudando bastante soluções relacionadas a infra e provisionamento. Sobre a cultura, tanto no profissional quanto no pessoal, pra mim tá sendo uma das melhores experiências já vivenciadas. Temos um processo de desenvolvimento muito bem definido porém vivo, onde qualquer um está apto p/ implementar melhorias. Ou seja, todos temos o mesmo nível e voz.

Além dessa experiência citada mais a cima, já trabalhei remoto em equipes 100% distribuídas, equipes assíncronas (sem definição de horário para iniciar o trabalho), equipes remotas onde o team inteiro era de uma cidade e a empresa de outra.

Para finalizar, sobre a maturidade do trabalho remoto no Brasil: Particularmente não tenho do que reclamar, em todas empresas que passei tive a oportunidade de trabalhar com ótimos profissionais além de um relacionamento incrível com cada empresa.

**Comment by renanlara** [comment]
Meu resumo:

- Salário completamente abaixo do mercado em contratação PJ independe de onde você mora.
- Sem benefício algum.
- Quando se precisa deslocar até a empresa, os gastos são seus, te vira.
- A empresa não ajuda em nada com custo para internet / máquina / software.

**Comment by mcarneiro** [comment]
@cjpatoilo era exatamente algo como esse texto seu que eu estava querendo :) não precisa responder cada uma das perguntas, mas compartilhar a experiência dessa forma mesmo. Valeu!

@renanlara pô cara, que merda. Mas confesso que vendo as vagas por aí que se dizem remotas era meio isso que eu tava esperando. Em poucas conversas que tive com amigos designers que já trabalharam nesse formato foi meio essa experiência aí. Em ambos os casos com empresas gringas.

**Comment by renanlara** [comment]
@mcarneiro, aqui no Brasil acredito que isso demore um bom tempo para ficar bacana para os profissionais. Eu não sei por que as empresas quer pagar o mesmo salário para um PJ igual para o CLT sendo que o CLT há um custo muito maior, ou seja, não repassa o valor de custo que teria o CLT.

Isso porque estamos falando somente de salário, sem falar dos demais custos do PJ remoto teria.

**Comment by felipefialho** [comment]
Bom, sou melhor respondendo perguntas do que falando livremente, hahahaha

- Existiu plano de carreira ou era só esquema "freela" / braço de desenvolvimento?
R: Era contratação efetiva, como se estivesse alocado na empresa.

- Ou melhor, alguém teve a experiência de ter uma equipe trabalhando remoto?
R: A equipe toda ficava remoto.

- A empresa dava ajuda de custo para internet / máquina / software?
R: Recebi acesso aos softwares da empresa (Pacote Adobe, 1Password, Sketch...)

- Relacionamento era 100% virtual ou existia eventos presenciais entre as pessoas de diferentes regiões? Se sim, imagino que empresa pagou passagem e estadia, certo?
R: Uma ou duas vezes por ano nos reuniamos para trabalhar alguns dias juntos.

- Alguém teve a experiência sendo contratado CLT?
R: Era PJ

- Salário era baseado no custo de vida do local do empregado ou da empresa? (o clássico conto do "uou, trabalho em Lençóis, no interior da Bahia com o salário de São Paulo!")
R: Era uma empresa norte-americana, mas recebia em reais, salário compatível com São Paulo, nem tanto em NYC. Ou seja, viveria muito bem em uma cidade menor, mas não tão bem no exterior.

- Haviam benefícios como saúde, alimentação, creche, etc? Se sim, como funcionavam? Ajuda de custo, convênio, etc?
R: Nada desse tipo.

- Existiu senso de equipe? Ou sentimento de que o time alocado tomava decisões sem consultar os remotos (já passei por isso trabalhando com uma empresa polonesa).
R: Todos participavam de todas as decisões.

Convoco o @wenderfreese, o @willianjusten, o @diogomoretti e o @mtmr0x para dar um alô.

**Comment by mcarneiro** [comment]
Valeu @LFeh, ajudou bastante!

**Comment by ribeirocodes** [comment]
Existiu plano de carreira ou era só esquema "freela" / braço de desenvolvimento?
R: Eu era PJ fixo, mas sem nenhum tipo de perspectiva de crescimento e/ou plano de carreira.

Ou melhor, alguém teve a experiência de ter uma equipe trabalhando remoto?
R: Todo o time de tech era remoto e na agência ficavam alocados os criativos e um gestor pra nossa equipe.

A empresa dava ajuda de custo para internet / máquina / software?
R: Nenhum, inclusive os softwares não eram licenciados. 

Relacionamento era 100% virtual ou existia eventos presenciais entre as pessoas de diferentes regiões? Se sim, imagino que empresa pagou passagem e estadia, certo?
R: 100% presencial, com uma péssima conduta de fazer os funcionários remotos ficarem 100% do tempo de trabalho com o hangout aberto e tela compartilhada (Sim, triste).

Salário era baseado no custo de vida do local do empregado ou da empresa? (o clássico conto do "uou, trabalho em Lençóis, no interior da Bahia com o salário de São Paulo!")
R: Com certeza com base do local do empregado, que inclusive é o motivo do time ser remoto, baratear custos. 

Haviam benefícios como saúde, alimentação, creche, etc? Se sim, como funcionavam? Ajuda de custo, convênio, etc?
R: Nope

Existiu senso de equipe? Ou sentimento de que o time alocado tomava decisões sem consultar os remotos (já passei por isso trabalhando com uma empresa polonesa).
R: Não, todas as decisões e demandas surgiam diretamente do gestor.

No geral mesmo com o trabalho exaustivo e condutas bem duvidáveis a agência reconhecia o esforço dos funcionários fornecendo auxílios financeiros quando solicitado e possíveis bônus quando a carga de trabalho era visivelmente excessiva.

**Comment by femontanha** [comment]
Fala ae Carneiro. Tive algumas experiências remotas, vou tentar responder.

Existiu plano de carreira ou era só esquema "freela" / braço de desenvolvimento?
- Plano de carreira é uma palavra muito forte, mas nenhuma das experiências que tive foi o esquema ed freela, era parte do time mesmo, contratado.

Ou melhor, alguém teve a experiência de ter uma equipe trabalhando remoto?
- Sim, e quando isso acontece geralmente é melhor. A cultura tem que ser muito forte de todo mundo se atualizar remotamente, senão só quem está remoto fica um pouco de lado.

A empresa dava ajuda de custo para internet / máquina / software?
- Quando era PJ não, quando fui CLT, deu máquina somente.

Relacionamento era 100% virtual ou existia eventos presenciais entre as pessoas de diferentes regiões? Se sim, imagino que empresa pagou passagem e estadia, certo?
- Existe eventos presenciais, sim a empresa que pagava os custos, geralmente era 1 semana de imersão

Alguém teve a experiência sendo contratado CLT?
Salário era baseado no custo de vida do local do empregado ou da empresa? (o clássico conto do "uou, trabalho em Lençóis, no interior da Bahia com o salário de São Paulo!")
- Sim já fui contratado como CLT, salário "normal" não diferente de quem estava alocado lá. 

Haviam benefícios como saúde, alimentação, creche, etc? Se sim, como funcionavam? Ajuda de custo, convênio, etc?
- Tive os benefícios basicos, refeicao, convenio médico, funciona normal pois era do mesmo estado, não sei como seriam pra outros estados.

Existiu senso de equipe? Ou sentimento de que o time alocado tomava decisões sem consultar os remotos (já passei por isso trabalhando com uma empresa polonesa).
Já aconteceu comigo em uma empresa, mas é questão de cultura, em um caso que tive o CTO tomou frente e começou a deixar tudo remoto.

Como citei, já trabalhei em times que só eu era remoto e já trabalhei em uma empresa que o time de dev inteiro era remoto, a segunda experiência foi muito melhor, mas tudo se resume a cultura do time.

**Comment by mcarneiro** [comment]
Valeu @ribeirovinicius!

Valeu @femontanha! O lance que eu pergunto de plano de carreira é basicamente se já viu alguém virar chefe, liderar projetos, tomar decisões e estar remoto com parte do time presencial, por exemplo. Foda pensar que são diversas variáveis possíveis. Tenho pensado na regra “se um está remoto todos estão remotos”, mas definitivamente é um desafio grande por ser bem cultural.
